intelmq/bots/BOTS

```diff
@@ -90,6 +90,14 @@
                 "rate_limit": 3600
             }
         },
+        "BitSight Ciberfeed Stream": {
+            "description": "Bitsight Collector is the bot reponsible to get Bitsight Ciberfeed Alert Stream",
+            "module": "intelmq.bots.collectors.bitsight.collector",
+            "parameters": {
+                "feed": "BitSight",
+                "http_url": "http://alerts.bitsighttech.com:8080/stream?key=<api>"
```

Comment:
Our feed is not https, yet.
As I said before I was just adding it for future compatibility. I, don't need it as of today.